app-modules/form/src/Filament/Resources/FormResource/Pages/EditForm.php

```diff
@@ -76,13 +78,14 @@ protected function getHeaderActions(): array
                             ->state(function (Form $form) {
                                 $code = resolve(GenerateSubmissibleEmbedCode::class)->handle($form);
 
-                                return <<<EOD
+                                $state = <<<EOD
                                 ```
                                 {$code}
                                 ```
                                 EOD;
+
+                                return new HtmlString(Str::markdown($state));
```

Comment:
```suggestion
                                return str($state)->markdown()->toHtmlString();
```

IMO this looks cleaner, you can apply to the other files too if you want?